training-service/src/app/content/course/component/course-list/course-list.component.scss

```diff
@@ -4,48 +4,48 @@
   margin-top: 1rem;
   display: flex;
   align-content: center;
+  flex-direction: row;
 }
-.check {
-  cursor: pointer
+
+.title {
+  display: block;
+  font-weight: bold;
 }
 
-.check input {
-  position: absolute;
-  top: 0;
-  left: 0;
-  visibility: hidden;
-  pointer-events: none
+.desc {
+  display: block;
 }
 
-.check span {
-  padding: 0.438rem 0.875rem;
-  display: inline-block;
-  text-transform: uppercase; 
-  width: 15rem; 
-  background-color: $button;
-  color: $white-color;
-  text-align: center;
-  font-weight: 450;
-  border-radius: 0.3rem;
-  
+.info {
+  display: block;
+  font-style: italic;
+  margin: 1rem 0rem 1rem 0rem;
+}
+.bold{
+  font-weight: 650;
 }
 
-.check input:checked+span {
- 
-  font-weight: 800 ;
-  text-decoration: underline;
-  background-color: $button;
+.list-group-item {
+  user-select: none;
 }
-.description{
-  font-weight: 400;
-  margin-top: 1rem;
-  color: $description;
 
+.list-group input[type="checkbox"] {
```

Comment:
Please do not use nested style selectors.
Please create class for input checkbox and use it.

Reply:
done